src/frontend/src/pages/index.js

```diff
@@ -40,12 +43,62 @@ export default function IndexPage() {
     getPosts(numPages);
   }, [telescopeUrl]);
 
+  useEffect(() => {
+    async function getPostsCount() {
+      try {
+        const res = await fetch('http://localhost:3000/posts');
+        // const res = await fetch(`${telescopeUrl}/posts`);
+        if (!res.ok) {
+          throw new Error(res.statusText);
+        }
+        return res.headers.get('x-total-count');
```

Comment:
If all you want is this header, you can change your `fetch()` so it does a `HEAD` request:

```js
fetch(url, {method: 'HEAD'})
```

We should confirm that our backend will support this, but it would be a lot lighter weight.

Reply:
Thanks for the suggestion! Confirmed it works.